doc/sphinx/changes.rst

```diff
@@ -8,6 +8,618 @@ Recent changes
+  (`#14452 <https://github.com/coq/coq/pull/14452>`_,
+  by Bart Jacobs).
+- **Changed:**
+  All occurrences of the name `CoqIde` to `CoqIDE`. This may cause issues with
+  installing and uninstalling desktop icons, causing apparent duplicates.
+  (`#14696 <https://github.com/coq/coq/pull/14696>`_, fixes `#14310
+  <https://github.com/coq/coq/issues/14310>`_, by Ali Caglayan).
+- **Added:**
+  Initial version of a visual debugger in CoqIDE.  Supports setting breakpoints
+  visually and jumping to the stopping point plus continue, step over,
+  step in and step out operations.  Displays the call stack and
+  variable values for each stack frame.  Currently only for Ltac.
+  See the documentation :ref:`here <coqide-debugger>`.
+  (`#14644 <https://github.com/coq/coq/pull/14644>`_,
+  fixes `#13967 <https://github.com/coq/coq/issues/13967>`_,
+  by Jim Fehrle)
+- **Fixed:**
+  It is now possible to deactivate the unicode completion
+  mechanism in CoqIDE
+  (`#14863 <https://github.com/coq/coq/pull/14863>`_,
+  by Pierre-Marie Pédrot).
+
+Standard library
+^^^^^^^^^^^^^^^^
+
+- **Changed:**
+  Permutation-related Proper instances are now at default priority instead of priority ``10``
+  (`#14574 <https://github.com/coq/coq/pull/14574>`_,
+  fixes `#14571 <https://github.com/coq/coq/issues/14571>`_,
+  by Gaëtan Gilbert).
+- **Changed:**
+  The new type of  `epsilon_smallest` is
+  `(exists n : nat, P n) -> { n : nat | P n /\ forall k, P k -> n <= k }`.
+  Here the minimality of `n` is expressed by `forall k, P k -> n <= k`
+  corresponding to the intuitive meaning of minimality
+  "the others are greater", whereas the previous version used
+  the negative equivalent formulation `forall k, k < n -> ~P k`.
+  Scripts using `epsilon_smallest` can easily be adapted using
+  lemmas `le_not_lt` and `lt_not_le` from the standard library.
+  (`#14601 <https://github.com/coq/coq/pull/14601>`_,
+  by Jean-Francois Monin).
+- **Removed:**
+  the file ``Numeral.v`` that was replaced by ``Number.v`` in 8.13.
+  (`#14819 <https://github.com/coq/coq/pull/14819>`_,
+  by Pierre Roux).
+- **Removed:**
+  some ``*_invol`` functions that were renamed ``*_involutive`` for consistency with the remaining of the stdlib in 8.13.
+  (`#14819 <https://github.com/coq/coq/pull/14819>`_,
+  by Pierre Roux).
+- **Deprecated:**
+  ``frexp`` and ``ldexp`` in `FloatOps.v`, renamed ``Z.frexp`` and ``Z.ldexp``
+  (`#15085 <https://github.com/coq/coq/pull/15085>`_,
+  by Pierre Roux).
+- **Added:**
+  A proof that incoherent equivalences can be adjusted to adjoint equivalences in ``Logic.Adjointification``.
+  (`#13408 <https://github.com/coq/coq/pull/13408>`_,
+  by Jasper Hugunin).
+- **Added:**
+  ``ltb`` and ``leb`` functions for ``string``, and some lemmas about them;
+  **Changed:**
+  ``ltb`` and ``leb`` functions for ``ascii``, into comparison-based definition
+  (`#14234 <https://github.com/coq/coq/pull/14234>`_,
+  by Yishuai Li).
```

Comment:
Another multi-entry single file to split.

Reply:
fixed